rubocop-default-config-ruby3.2.yml

```
@@ -0,0 +1,9 @@
+inherit_from:
+  - https://raw.githubusercontent.com/wtag/default-configs/master/rubocop-default-config-ruby2.6.yml
+AllCops:
+  Exclude:
+    - "vendor/**/*"
+    - "db/schema.rb"
+  UseCache: false
+  TargetRubyVersion: 3.2
+  NewCops: enable
```

As it inherits from 2.6, I think this can be further simplified:
```suggestion
  TargetRubyVersion: 3.2
```